### DISCUSSION TITLE
Docs sidebar should not require 5 clicks to get to an arbitrary page

### DISCUSSION THREAD

**Discussion by jasikpark** [dubzzz/fast-check#3825] [Ideas]
<img width="362" alt="image" src="https://user-images.githubusercontent.com/10626596/236886860-c0a32de4-ea1b-4340-93ea-2787d9cfce23.png">

It would be great if I could click on the top-level header and see all the nested pages immediately, rather than clicking on each section individually - it makes it harder to scan what's all available

I'm pretty sure Docusaurus has support for non-collapsible headings? I thought https://docs.defined.net/api/host-create/ used it, but I guess I was wrong

**Comment by dubzzz** [comment]
Good point! I'll try to find a way to make it easier to browse.

So far, I'm not convinced with the content I have in my _folder_ pages. I'm trying to find a way to make these pages more useful (or drop them at some point).

**Reply by jasikpark** [reply]
thx!

**Reply by jasikpark** [reply]
also bravo on the docs! excited to see these exist 😄
